repo/packages/Z/zeppelin/7/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "packagingVersion": "4.0",
+    "name": "zeppelin",
+    "version": "0.7.3-3-2.2.1",
```

Comment:
Minor: I would just add a package version here in case you want to update while keeping zeppelin and hadoop version the same